src/result_tests.rs

```diff
@@ -81,3 +81,10 @@ pub fn tests_get_cql_wrapper_tiny_int() -> CqlValueWrapper {
     let element = CqlValue::TinyInt(3);
     CqlValueWrapper { inner: element }
 }
+
+#[napi]
+/// Test function returning sample CqlValueWrapper with UUID type
```

Comment:
Are you sure this is UUID type? Ah those pesky comments while copying and pasting.